domain/src/main/java/com/github/maleksandrowicz93/educational/institution/FacultySetup.java

```diff
@@ -0,0 +1,13 @@
+package com.github.maleksandrowicz93.educational.institution;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+
+import java.util.Set;
+
+@Builder(access = AccessLevel.PACKAGE)
```

Comment:
nie będzie domyślnie jako package, jeśli record jest package? (analogiczny komentarz w podobnych miejscach)

Reply:
Zdelombokowałem domyślnego buildera i okazał się public. Oczywiście jeśli klasa jest package private, to i tak do tego buildera nie będzie dostępu, jednak jeśli chcę mieć klasę package private, to po prostu chcę uniknąć posiadania czegokolwiek, co jest public. Lubię podejście dodawania słówka public wtedy i tylko wtedy gdy faktycznie dany konstrukt zostaje przeznaczony do udostępnienia światu.